docs/process/teams.md

```diff
@@ -0,0 +1,29 @@
+# Teams
+
+**Who do you `@` for what? See below!**
+
+_(These teams are not set in stone and will likely evolve as we continue to iterate on what makes the most sense for the success of the product.)_
+
+## General Teams
+
+These are good teams to start with for general communication and questions. (Members of these teams are also members of the skills-based teams below.)
+
+| Team | Purpose |
+|:--|:--|
+| `@desktop/maintainers` | The people designing, developing, and driving GitHub Desktop |
+| `@desktop/comrades` | Community members with a track record of activity in the Desktop project |
+| `@desktop/contributors` | Community members active in the Desktop project |
+
+## Skills-based Teams
```

Comment:
Specialized Teams? I have skills too 😭 lol

Reply:
oh no! hahaha. 💜 

my thinking is that eventually (hopefully soon?) we will also add comrades to skills-based teams!

Reply:
Just trying to get that final wordsmithing polish you know? I honestly dont care just thought it was a  good joke opportunity.